pixi.toml

```diff
@@ -121,7 +124,7 @@ pytest-xdist = ">=3.6.1"
 typeguard = ">=4.3.0"
 
 # Dynamixel SDK for LEAP hand
-dynamixel-sdk = ">=3.7"
+dynamixel-sdk = ">=3.7.0"
```

Comment:
i would actually move the dynamixel dependency out of this dependency group (which is meant for testing). you can create a new dependency group called "leap" (do this by defining a new toml header `[feature.leap.pypi-dependencies]`) and then putting `dynamixel-sdk` under it. You should then make sure `dev` and `dev-no-gpu` include the `leap` dependency group.